kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/ClusterService.java

```diff
@@ -272,5 +274,11 @@ private KafkaCluster updateCluster(String topicToDelete, String clusterName,
         .flatMap(offsets -> kafkaService.deleteTopicMessages(cluster, offsets));
   }
 
+  public Mono<PartitionsIncreaseResponse> increaseTopicPartitions(
+      String clusterName,
+      Mono<PartitionsIncrease> partitionsIncrease) {
+    return clustersStorage.getClusterByName(clusterName).map(cluster ->
+        kafkaService.increaseTopicPartitions(cluster, partitionsIncrease)).orElse(Mono.empty());
```

Comment:
It may cause issues of you will return empty mono, it's better to throw error if there is no such cluster